pyproject.toml

```diff
@@ -30,32 +26,20 @@ path = "src/sknnr/__about__.py"
 packages = ["src/sknnr"]
 
 [tool.hatch.build.targets.sdist]
-include = [
-    "/src",
-]
+include = ["/src"]
 
 [tool.hatch.envs.default]
-dependencies = [
-    "pre-commit",
-]
+dependencies = ["pre-commit"]
 
 [tool.hatch.envs.test]
-dependencies = [
-    "pytest",
-    "pytest-cov",
-    "pandas"
-]
+dependencies = ["pytest", "pytest-cov", "pandas"]
 
 [tool.hatch.envs.test.scripts]
```

Comment:
You could add `--doctest-modules` to the test scripts while you're making updates, but I don't think there are any doctests to run, so it would just be pre-emptive. Your call!

Reply:
Good call!  I've added that to `pyproject.toml`